ui/src/audioplayer/PlayerToolbar.js

```diff
@@ -4,23 +4,28 @@ import { useGetOne } from 'react-admin'
 import { GlobalHotKeys } from 'react-hotkeys'
 import { LoveButton, useToggleLove } from '../common'
 import { keyMap } from '../hotkeys'
+import { ThemeProvider } from '@material-ui/styles'
+import { createMuiTheme } from '@material-ui/core/styles'
+import useCurrentTheme from '../themes/useCurrentTheme'
+import { QualityInfo } from '../common/QualityInfo'
 import config from '../config'
 
 const Placeholder = () =>
   config.enableFavourites && <LoveButton disabled={true} resource={'song'} />
 
 const Toolbar = ({ id }) => {
   const location = useLocation()
+  const theme = useCurrentTheme()
   const resource = location.pathname.startsWith('/song') ? 'song' : 'albumSong'
   const { data, loading } = useGetOne(resource, id)
   const [toggleLove, toggling] = useToggleLove(resource, data)
 
   const handlers = {
     TOGGLE_LOVE: useCallback(() => toggleLove(), [toggleLove]),
   }
-
   return (
-    <>
+    <ThemeProvider theme={createMuiTheme(theme)}>
```

Comment:
Please remove this. I removed in a recent commit (16e495a80fd791fad0b4c9053c109cf831711a69) to fix a bug, and this is adding it back....